spec/factories/events.rb

```diff
@@ -8,4 +8,8 @@
     name { SecureRandom.alphanumeric }
     entity_id { 'https://test-entity-id' }
   end
+
+  factory :new_team_event do
+    name { 'fake team name' }
```

Comment:
`GDS United`
or
`Team Awesome`
or
`The Rotatorators`
or
`Gryffindor`

😛

Reply:
Thanks for the fun suggestions i'll go for the more conservative Team Awesome